onnxruntime_extensions/tools/pre_post_processing/steps/nlp.py

```diff
@@ -0,0 +1,449 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import onnx
+from collections import OrderedDict
+
+from typing import List, Optional, Tuple, Union
+from ..step import Step
+
+
+class SentencePieceTokenizer(Step):
```

Comment:
Need to add unit tests to \test\test_tools_add_pre_post_processing_to_model.py

Reply:
Done.